sympy/functions/elementary/tests/test_piecewise.py

```diff
@@ -495,3 +500,19 @@ def test_S_srepr_is_identity():
     p = Piecewise((10, Eq(x, 0)), (12, True))
     q = S(srepr(p))
     assert p == q
+
+
+def test_issue_10258():
+    assert Piecewise((0, x < 1), (1, True)).is_zero is False
```

Comment:
note to self: perhaps better to return None (if possible) keep old code but instead of return when_multiple, return None (and keep the Expr header test).